test/serialize-deserialize-test.coffee

```diff
@@ -0,0 +1,170 @@
+{assert} = require('chai')
+sinon = require('sinon')
+
+blueprintApi = require('../src/blueprint-api')
+
+
+describe('Serialization of the Blueprint interface (Application AST)', ->
+  describe('Blueprint object', ->
```

Comment:
I think what kubula wanted here is testing the Blueprint Object which can be done with refract too IIRC.